src/jit-format/jit-format.cs

```diff
@@ -104,7 +104,8 @@ private void SetPlatform()
                         }
                         else
                         {
-                            _os = match.Groups[1].Value.Trim();
+                            // Assuming anything other than Windows or OSX is a Linux flavor
+                            _os = "Linux";
```

Comment:
That sounds fine.  My main gripe is a case being used as a catch-all when we know it's not.  We should match what we know we want to support and gracefully error out on other cases, even if there's a chance the catch-all would actually do the right thing.